.github/workflows/lint.yml

```diff
@@ -0,0 +1,32 @@
+name: Lint
+
+on: [push]
+
+jobs:
+  ESLint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v1
+      - name: Set up Node 10.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+      - name: Set up ESLint
+        run: npm install --global eslint
+      - name: Run ESLint checks
+        run: eslint .
+
+  Prettier:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v1
+      - name: Setup up Node 10.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+      - name: Set up Prettier
+        run: npm install --global prettier
+      - name: Run Prettier checks
+        run: prettier -c "**/*.{md,scss,js,json}"
```

Comment:
As we are reading `md` files, is prettier also pretifying html? If so, it could be interesting to also parse the `njk` files in the future.

Reply:
I took a look at it, but it seems there is no a reliable nunchucks prettier plugin currently available. The only one I've managed to find is [justrhysism/prettier-plugin-nunjucks](https://github.com/justrhysism/prettier-plugin-nunjucks) (via this [issue](https://github.com/prettier/prettier/issues/5754) in the Prettier repository), but it doesn't seem like it's ready for prime time yet.

I've found a functional VS Code plugin though: [Nunjucks Template Formatter](https://marketplace.visualstudio.com/items?itemName=okitavera.vscode-nunjucks-formatter).

Reply:
no big deal them, there will be more `md` files than `njk` ones.